doc/docs/master/reference/scaling_limits.md

```diff
@@ -0,0 +1,56 @@
+# Pipeline Scaling Limits in Community Edition
+
+Pachyderm Community Edition **limits the number of pipelines** users can deploy and **caps the parallelism** settings of those pipelines.  
```

Comment:
Our completely free Pachyderm Community Edition contains built-in scaling limitations and parallelism thresholds. If you want to scale beyond these limits, request your [FREE 30-Day Pachyderm Enterprise token](https://www.pachyderm.com/trial) today and enjoy unlimited scaling, JupyterHub integration, and more!

You can also enjoy unlimited scaling on [Pachyderm Hub](https://hub.pachyderm.com) if wrangling infrastructure isn't your thing.

Reply:
Found a common ground with:
Our free Pachyderm Community Edition contains built-in scaling limitations and parallelism thresholds. If you want to scale beyond these limits, request your FREE 30-Day Pachyderm Enterprise token and enjoy unlimited scaling, JupyterHub integration, and more.